production-deployment-using-tiup.md

```diff
@@ -114,13 +114,13 @@ aliases: ['/docs-cn/dev/how-to/deploy/orchestrated/tiup/']
 {{< copyable "shell-regular" >}}
 
 ```shell
-tiup cluster deploy tidb-test v4.0.0-rc.2 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
+tiup cluster deploy tidb-test v4.0.0 ./topology.yaml --user root [-p] [-i /home/root/.ssh/gcp_rsa]
 ```
 
 以上部署命令中：
 
 - 通过 TiUP cluster 部署的集群名称为 `tidb-test`
-- 部署版本为 `v4.0.0-rc.2`，其他版本可以执行 `tiup list tidb --refresh` 来查看 TiUP 支持的版本
+- 部署版本为 `v4.0.0`，最新版本可以通过执行 `tiup list tidb --refresh` 来查看 TiUP 支持的版本
```

Comment:
```suggestion
- 部署版本为 `v4.0.0`，最新版本可以通过执行 `tiup list tidb` 来查看 TiUP 支持的版本
```
The --refresh has been removed at TiUP v1.0.0 according to https://github.com/pingcap/docs-cn/pull/3417.

FYI @kissmydb @lonng